### ISSUE TITLE
Could you provide an example code about mnist? I am comfused with the usage of DataParallelCriterion.

### ISSUE THREAD

**Issue by weitaoatvison** [zhanghang1989/PyTorch-Encoding#112]
Hi, Could you provide an example code about mnist? It seems you have provide the code before but now it lost. I am comfused with the usage of DataParallelCriterion. I guess when using sync-bn need to customize the loss layer?

**Comment by weitaoatvison** [comment]
@zhanghang1989

**Comment by zhanghang1989** [comment]
The example is in the docs https://hangzhang.org/PyTorch-Encoding/parallel.html#dataparallelcriterion
```python
>>> net = encoding.nn.DataParallelModel(model, device_ids=[0, 1, 2])
>>> criterion = encoding.nn.DataParallelCriterion(criterion, device_ids=[0, 1, 2])
>>> y = net(x)
>>> loss = criterion(y, target)
```

**Comment by weitaoatvison** [comment]
Thanks. I have tested the mnist myself, it seems work well. But I found it seems has something wrong  in pytorch-encoding. I think it may caused by code update.
